crates/re_renderer/src/size.rs

```diff
@@ -68,15 +67,15 @@ impl std::ops::Mul<f32> for Size {
 
     #[inline]
     fn mul(self, rhs: f32) -> Self::Output {
-        debug_assert!(rhs.is_finite() && rhs >= 0.0);
+        debug_assert!(rhs.is_normal() && rhs >= 0.0);
```

Comment:
This seems to forbid multiplication with zero: https://doc.rust-lang.org/std/primitive.f32.html#method.is_normal

```suggestion
        debug_assert!(0.0 <= rhs && rgb <= f32::MAX);
```

note though that multiplying with anything >1.0 can result in over-flow to infinity

Reply:
argh. how is 0.0 not normal T_T